nbextensions/bulkImportCell/actionButton.js

```diff
@@ -5,7 +5,8 @@ define([
 ) => {
     'use strict';
 
-    const div = html.tag('div');
+    const div = html.tag('div'),
+        cssCellType = 'kb-bulk-import';
```

Comment:
I just want to note that we may want to consider making this widget generic, and use it for the bulk analysis cell (and import CSV, maybe) as well. So maybe the cell type should be an option?

Reply:
@briehl my ticket DATAUP-304 is making the action button generic.

Reply:
Yup -- ideally it'd come from introspecting some aspect of the cell, rather than having it hard-coded. I'm fairly sure that a number of these components do not really need the cell type info anyway, as they would appear the same in an app cell, a bulk analysis cell, or a bulk import.

Reply:
Alrighty. Let's just leave it for now.